node/node.d.ts

```diff
@@ -965,16 +965,37 @@ declare module "child_process" {
         stdio?: any;
         customFds?: any;
         env?: any;
+        encoding: void;
```

Comment:
ah, I understand what you need. I'll make a another pull request. wait a moment.